### PR TITLE
[MIG][10.0] stock picking show backorder

### PR DESCRIPTION
**Pull request by AaronHForgeFlow**
Plug & Play migration

### CODE REVIEW

**Review by Cedric-Pigeon — APPROVED** [review]
Code review

File: stock_picking_show_backorder/models/stock.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# -*- coding: utf-8 -*-
+# Copyright 2015-17 Eficent Business and IT Consulting Services, S.L.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from openerp import fields, models
```
Comment (LoisRForgeFlow) [inline stock_picking_show_backorder/models/stock.py]: odoo

**Comment by lmignon** [comment]
@aheficent Can you take into account the comment from @lreficent?

**Review by zakiuu — APPROVED** [review]
Code review

**Comment by pedrobaeza** [comment]
Please squash your last 2 commits and I'll merge.

**Comment by AaronHForgeFlow** [comment]
I think I messed up here with the history. Don't merge it yet.

**Comment by LoisRForgeFlow** [comment]
@aheficent is this now ready?

**Comment by AaronHForgeFlow** [comment]
@lreficent Yes it is :ok: 
Update: No it's not :face_with_head_bandage:

**Comment by AaronHForgeFlow** [comment]
This is as it was before I messed up with the history. Runbot was killed for other reason. IMO this can be merged @pedrobaeza

**Comment by pedrobaeza** [comment]
OK, merging

**Pull request merged**